pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -177,21 +175,12 @@ var _ = Describe("Pod Network", func() {
 		mockNetwork.EXPECT().NftablesAppendRule("nat", "prerouting", "iifname", "eth0", "counter", "jump", "KUBEVIRT_PREINBOUND").Return(nil).AnyTimes()
 		mockNetwork.EXPECT().NftablesAppendRule("nat", "postrouting", "oifname", "k6t-eth0", "counter", "jump", "KUBEVIRT_POSTINBOUND").Return(nil).AnyTimes()
 
-		err := SetupPodNetwork(vm, domain)
+		err := SetupPodNetworkPhase1(vm)
 		Expect(err).To(BeNil())
-		Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(1))
-		xmlStr, err := xml.Marshal(domain.Spec.Devices.Interfaces)
-		Expect(string(xmlStr)).To(Equal(string(interfaceXml)))
-		Expect(err).To(BeNil())
-
-		// Calling SetupPodNetwork a second time should result in no
-		// mockNetwork function calls and interface should be identical
-		err = SetupPodNetwork(vm, domain)
 
-		Expect(err).To(BeNil())
-		Expect(len(domain.Spec.Devices.Interfaces)).To(Equal(1))
-		xmlStr, err = xml.Marshal(domain.Spec.Devices.Interfaces)
-		Expect(string(xmlStr)).To(Equal(string(interfaceXml)))
+		// Calling SetupPodNetworkPhase1 a second time should result in no
+		// mockNetwork function calls
```

Comment:
How do you check that this is indeed the case?

Reply:
I've limited the number of times mocked entry points can be called. Let me know if that's what you are looking for.